test/settings/settings_test.go

```diff
@@ -0,0 +1,49 @@
+package settings_test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/envoyproxy/ratelimit/src/settings"
+)
+
+func TestShouldReturnDefaultValueForRedisPipelineWindow(t *testing.T) {
+	expectedValue := time.Duration(0) * time.Second
+
+	s := settings.NewSettings()
+	if s.RedisPipelineWindow != expectedValue {
+		t.Errorf("Got %s, expected %s", s.RedisPipelineWindow, expectedValue)
+	}
```

Comment:
Please use testing assertions like the other tests.

Reply:
Hi @mattklein123 I've updated the test for the settings package to use the test suite by github.com/stretchr/testify.

Please kindly review. Thank you.